update/stock_update.js

```diff
@@ -20,27 +19,60 @@ function findStockDatum(stocks, company) {
   return undefined;
 }
 
+function sortArticles(articles) {
+  
+  articles.sort(function(a, b) {
+    return new Date(b.date) - new Date(a.date);
+  });
+  
+  return articles;
+}
+
+function articleContains(article, articles) {
+  for (var x=0; x<articles.length; x++) {
+    if (article.url === articles[x].url) {
+      return true;
+    }
+  }
+  return false;
+}
+
```

Comment:
checkout https://www.npmjs.com/package/sort-by

Reply:
That looks cool. May not want to add another dependency for one sort function, but might not matter. If we sorted more things, definitely useful.